setup.py

```diff
@@ -17,56 +17,66 @@
 from setuptools import setup, find_packages
 
 def read(*rnames):
-    return open(os.path.join(os.path.dirname(__file__), *rnames)).read()
+    with open(os.path.join(os.path.dirname(__file__), *rnames)) as f:
+        return f.read()
 
-setup (
+CHAMELEON_REQUIRES = [
+    'z3c.pt >= 2.1',
+    'z3c.ptcompat>=1.0',
```

Comment:
I wish you'd pick one: spaces around `>=` or no spaces around `>=`.

Reply:
In my defence, I didn't write that. [I just moved it](https://github.com/zopefoundation/z3c.template/pull/2/files#diff-2eeaed663bd0d25b7e608891384b7298L65) from lower in the file.

Reply:
Apologies for mis-assigning the blame!